usaspending_api/disaster/v2/views/elasticsearch_base.py

```diff
@@ -90,7 +90,9 @@ def post(self, request: Request) -> Response:
             non_zero_queries.append(ES_Q("range", **{field: {"lt": 0}}))
         self.filter_query.must.append(ES_Q("bool", should=non_zero_queries, minimum_should_match=1))
 
-        self.bucket_count = get_number_of_unique_terms_for_awards(self.filter_query, f"{self.agg_key}.hash")
+        self.bucket_count = get_number_of_unique_terms_for_awards(
+            self.filter_query, f"{self.agg_key.replace('.keyword', '')}.hash"
```

Comment:
Minor concern here since it seems to alter code used by many endpoints, not only the CFDA endpoints. A brief examination of the code makes it look OK, but asking for confirmation this was thoroughly tested

Reply:
This won't effect anything. Was a way around adding another conditional to this page since none of the other "agg_keys" are going to have `.keyword` in them since they are a `keyword` type.